libraries/radwordsprocessing/formats-and-conversion/word-file-formats/docx/settings.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Settings
 page_title: Settings
-description: Settings
+description: This article describes the import/export settings which provide additional modification options
```

Comment:
This article describes the settings that you can use to control how the DOCX documents are imported/exported in WordsProcessing.